plugin/iamutil/resource_parser.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) HashiCorp, Inc.
 // SPDX-License-Identifier: MPL-2.0
 
-//go:generate go run internal/generate_resources.go
+//go:generate go run github.com/hashicorp/vault-plugin-secrets-gcp/plugin/iamutil/internal/
```

Comment:
It doesn't look like this ever worked 🤔 Upon upgrading to Go 1.21+, the inclusion of the toolchain directive seems to have uncovered this issue. The incorrect path was breaking tests once the Go version was upgraded, and after meeting with @tomhjp, we decided that using the full path to the underlying directory is the better way to generate and import these resources.